tests/components/deconz/test_config_flow.py

```diff
@@ -458,63 +464,59 @@
     gateway = await setup_deconz_integration(hass)
 
     result = await hass.config_entries.flow.async_init(
-        config_flow.DOMAIN,
+        DOMAIN,
         data={
-            config_flow.CONF_HOST: "2.3.4.5",
-            config_flow.CONF_PORT: 8080,
-            config_flow.CONF_API_KEY: "updated",
-            config_flow.CONF_SERIAL: BRIDGEID,
+            CONF_HOST: "2.3.4.5",
+            CONF_PORT: 8080,
+            CONF_API_KEY: "updated",
+            CONF_SERIAL: BRIDGEID,
         },
         context={"source": "hassio"},
     )
 
-    assert result["type"] == "abort"
+    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "already_configured"
-    assert gateway.config_entry.data[config_flow.CONF_HOST] == "2.3.4.5"
-    assert gateway.config_entry.data[config_flow.CONF_PORT] == 8080
-    assert gateway.config_entry.data[config_flow.CONF_API_KEY] == "updated"
+    assert gateway.config_entry.data[CONF_HOST] == "2.3.4.5"
+    assert gateway.config_entry.data[CONF_PORT] == 8080
+    assert gateway.config_entry.data[CONF_API_KEY] == "updated"
 
 
 async def test_hassio_discovery_dont_update_configuration(hass):
     """Test we can update an existing config entry."""
     await setup_deconz_integration(hass)
 
     result = await hass.config_entries.flow.async_init(
-        config_flow.DOMAIN,
+        DOMAIN,
         data={
-            config_flow.CONF_HOST: "1.2.3.4",
-            config_flow.CONF_PORT: 80,
-            config_flow.CONF_API_KEY: API_KEY,
-            config_flow.CONF_SERIAL: BRIDGEID,
+            CONF_HOST: "1.2.3.4",
+            CONF_PORT: 80,
+            CONF_API_KEY: API_KEY,
+            CONF_SERIAL: BRIDGEID,
         },
         context={"source": "hassio"},
     )
 
-    assert result["type"] == "abort"
+    assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "already_configured"
 
 
 async def test_option_flow(hass):
     """Test config flow options."""
-    entry = MockConfigEntry(domain=config_flow.DOMAIN, data={}, options=None)
-    hass.config_entries._entries.append(entry)
+    gateway = await setup_deconz_integration(hass)
 
-    flow = await hass.config_entries.options.async_create_flow(
-        entry.entry_id, context={"source": "test"}, data=None
-    )
+    result = await hass.config_entries.options.async_init(gateway.config_entry.entry_id)
 
-    result = await flow.async_step_init()
-    assert result["type"] == "form"
+    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5028567)